src/widgets/animimage/lv_animimage.c

```diff
@@ -161,6 +161,11 @@ static void index_change(lv_obj_t * obj, int32_t index)
     int32_t idx;
     lv_animimg_t * animimg = (lv_animimg_t *)obj;
 
+    if(animimg->dsc == NULL) {
+        LV_LOG_ERROR("dsc is null");
```

Comment:
ERROR is used for crash or close-crash situations. I think it should be just a warning.

Reply:
Yes, it should be warning